scipy/stats/morestats.py

```diff
@@ -1057,17 +1061,21 @@ def boxcox(x, lmbda=None, alpha=None):
         return y, lmax, interval
 
 
-def boxcox_normmax(x, brack=(-2.0, 2.0), method='pearsonr'):
+def boxcox_normmax(x, brack=None, method='pearsonr', bounds=None,
+                   optimize_method='brent', tol=None, options=None):
     """Compute optimal Box-Cox transform parameter for input data.
 
     Parameters
     ----------
     x : array_like
         Input array.
-    brack : 2-tuple, optional
-        The starting interval for a downhill bracket search with
-        `optimize.brent`.  Note that this is in most cases not critical; the
-        final result is allowed to be outside this bracket.
+    brack : sequence, optional
+        For `optimize_method` 'brent' and 'golden', `bracket` defines the bracketing
+        interval and can either have three items ``(a, b, c)`` so that
+        ``a < b < c`` and ``fun(b) < fun(a), fun(c)`` or two items ``a`` and
+        ``c`` which are assumed to be a starting interval for a downhill
+        bracket search; it doesn't always mean that the obtained solution will
+        satisfy ``a <= x <= c``. See `scipy.optimize.minimize_scalar` for more details.
```

Comment:
The sentence is a bit too long. Consider 
* making a bulleted list for the two options
* hard stop instead of the semicolon.  
* Replace "it doesn't always mean that the obtained solution ..." with "The solution will not necessarily satisfy ...; see `scipy.optimize.minimize_scalar`..."  
* In ``a <= x <= c``, isn't ``x`` really lambda?